### ISSUE TITLE
DragOverlay - Small conflict issue

### ISSUE THREAD

**Issue by rasa2k** [Viglino/ol-ext#230]
Hi, 

When using closeBox: true on the PopUp Object, the drag propagation conflict with the close/remove overlay. I've tried to fix it with css but without any luck.  KR

**Comment by Viglino** [comment]
Hi,

The drag overlay was not intended to be used with a close box or with any action in the popup (link, button, etc.) as the event will propagate on the map.
I'll try to see what can be done to stop event propagation on click on the close box.

@+

**Comment by rasa2k** [comment]
Yeah that’s what I thought, but I was hoping it would. I also tried going with the PopUpFeature. With that, I can at least use the canFix option, but the downside is the template option. It dictates’ at specific template which is not helping me. Can it be overwritten?
Maybe you have some other modules that fit my requirements a bit more? What I’m really after is a popup/overlay that can be dragged, has a close button and with clickable content. 
KR,

**Comment by Viglino** [comment]
Hi,
The PopUpFeature is specific and use a feature to show the feature attributes.
I think it will be possible to have the close box button working with the drag interaction. 
The idea is to enable clicks on buttons and links (`<button>`, `<a>`) but let drag the popup when over other elements.
I'll try to commit asap.
@+

**Comment by Viglino** [comment]
Last commit lets you add a close box on popup when dragging.
The buttons and links (<button>, <a>) in the popup are now clickable, just click an other element to move the popup.
